RELEASE/scripts/autoscend.ash

```diff
@@ -13361,13 +13361,23 @@ boolean auto_tavern()
 	{
 		abort("Quest not yet started, talk to Bart Ender and re-run.");
 	}
+
+	// Use Tavern Text to get the finished message instead of relying on Mafia
+	string temp = visit_url("tavern.php?place=barkeep");
+	if(contains_text(temp, "Thanks again for taking care of that rat problem"))
+	{
+		set_property("auto_tavern", "finished");
+		return true;
+	}
+
 	# Mafia usually fixes tavernLayout when we visit the cellar. However, it sometimes leaves it in a broken state so we can't guarantee this will actually help. However, it will result in no net change in tavernLayout so at least we can abort.
-	string tavern = get_property("tavernLayout");
+/*	string tavern = get_property("tavernLayout");
```

Comment:
Remove code rather than commenting out. We have git history if we need to recover it.

Reply:
Changed